### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by jamiebuilds** [yarnpkg/website#1]
Over the weekend I built out most of the Yarn website and it's now ready for a push to fill out the documentation.

We have a bunch of people working on the documentation in a war room in 14.03 by the walkway to building 12, on September 27-28, so come by if you'd like to help!
## TODO

Here are all the pages of documentation needed before publish. Everything is broken down into "guides". The guides are each accessible from the /en/docs/ page.
- Getting Started
  - [x] Getting Started (@thejameskyle)
  - [x] Installation (@thejameskyle)
  - [x] Usage (@thejameskyle)
- The Yarn Workflow (see https://github.com/yarnpkg/website/pull/37)
  - [x] Creating a package.json (including common things) (@JoelMarcey)
  - [x] Installing dependencies (@JoelMarcey)
  - [ ] Running scripts (@JoelMarcey)
  - [x] Updating dependencies (including details on lockfile) (@JoelMarcey)
- CLI Commands
  - [x] Intro (not taken)
  - [ ] yarn access (not taken) -- not implemented yet
  - [x] yarn add (@lacker)
  - [x] yarn cache (not taken) -- not implemented yet
  - [x] yarn check (@ericnakagawa)
  - [x] yarn clean (@ericnakagawa)
  - [ ] yarn config (not taken) (some functionality on `missing-commands` branch)
  - [x] yarn dist-tag (@thejameskyle)
  - [x] yarn global (not taken)
  - [x] yarn info (@jamesgpearce, https://github.com/yarnpkg/website/pull/28)
  - [x] yarn init (not taken)
  - [x] yarn install (@thejameskyle)
  - [x] yarn licenses (@thejameskyle)
  - [ ] yarn link (not taken) -- couldn't test, may be unimplemented https://github.com/yarnpkg/yarn/issues/336
  - [x] yarn login (@thejameskyle)
  - [x] yarn logout (@thejameskyle)
  - [x] yarn ls (@thejameskyle)
  - [x] yarn outdated (@ericnakagawa) (on `missing-commands` branch)
  - [x] yarn owner (@lacker)
  - [x] yarn pack (@ericnakagawa)
  - [x] yarn publish (@lacker)
  - [x] yarn run (not taken) -- the set of scripts is not finalized
  - [ ] yarn self-update (not taken) -- on hold until 'config' added
  - [x] yarn team (not taken) -- no code yet, need to review PR
  - [x] yarn uninstall (@lacker)
  - [x] yarn unlink (not taken)
  - [x] yarn upgrade (@caabernathy) --waiting on this issue resolution https://github.com/yarnpkg/yarn/issues/458
  - [x] yarn version (@jamesgpearce, https://github.com/yarnpkg/website/pull/46)
  - [x] yarn why (@jamesgpearce, https://github.com/yarnpkg/website/pull/43)
- Configuring Yarn (still needs a placeholder)
  - [x] The package.json format (@ericvicenti)
  - [x] The yarn.lock format (@ericvicenti)
  - [ ] Folders? (not taken)
- Migrating from npm
  - [ ] Migrating from npm (@lacker)
    - Include info about shrinkwraps (yarnpkg/yarn#397)
- Creating a Package
  - [x] Creating a package (@JoelMarcey) (see https://github.com/yarnpkg/website/pull/29)
  - [x] Preparing a package for publish (@JoelMarcey) (see https://github.com/yarnpkg/website/pull/40)
  - [x] Publishing a package (@JoelMarcey) (see https://github.com/yarnpkg/website/pull/42)
- How Yarn Works
  - [ ] How Yarn Works (not taken)
- The Yarn Organization (@thejameskyle)
#### Misc Tasks
- [ ] Go through again and update documentation to include details about alternative registries
- [ ] Review documentation for 

Many of these line up with documents on the npm docs website: https://docs.npmjs.com/, please don't just copy and paste them. There are a number of things that are different in Yarn, things that haven't been implemented yet, things that npm has for legacy reasons, etc.

When writing documentation, make sure that you're writing them for a reader that has never used npm before. Don't require prior knowledge about how npm works and don't send people off to the npm documentation unless you're discussing interop with npm directly.

---
## Tips

I wrote up a guide on (what I think are) best practices when writing documentation: https://github.com/thejameskyle/documentation-handbook

Remember that you must follow the Yarn Code of Conduct: https://github.com/yarnpkg/yarn/blob/master/CODE_OF_CONDUCT.md (Contributor's Covenant)

---
## Before you go

If you want to take one or more of these docs on please update the issue with your **@username** so others know you're working on it.
